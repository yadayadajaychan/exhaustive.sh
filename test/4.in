3A 1B
