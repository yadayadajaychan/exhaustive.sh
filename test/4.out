A=3'd0; B=1'd0; #10;
A=3'd0; B=1'd1; #10;
A=3'd1; B=1'd0; #10;
A=3'd1; B=1'd1; #10;
A=3'd2; B=1'd0; #10;
A=3'd2; B=1'd1; #10;
A=3'd3; B=1'd0; #10;
A=3'd3; B=1'd1; #10;
A=3'd4; B=1'd0; #10;
A=3'd4; B=1'd1; #10;
A=3'd5; B=1'd0; #10;
A=3'd5; B=1'd1; #10;
A=3'd6; B=1'd0; #10;
A=3'd6; B=1'd1; #10;
A=3'd7; B=1'd0; #10;
A=3'd7; B=1'd1; #10;
